Assets/Scripts/UI/HUD/WormthBar.cs

```diff
@@ -0,0 +1,15 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+public class WormthBar : MonoBehaviour
+{
+    public Wormth wormth;
+    public Scrollbar bar;
+
+    private void Update()
+    {
+        bar.size = wormth.heat / 100;
```

Comment:
100 shouldn't be hardcoded, and it should be configurable using the inspector. Add a field in `Wormth` and call it `maxHeat`. Use that field here